src/components/Carousel.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Carousel = props => {
+  return typeof props.images === 'object' && props.images.length ? (
+    <div>
+      {props.images.map(image => {
```

Comment:
```suggestion
           {props.images.map(image => 
               <img key={image.id} src={image.url} alt={image.description} />
      )}
```

Equivalent but feels so good